ecosystem/sep-0006.md

```diff
@@ -301,9 +359,91 @@ const result = await this.server.submitTransaction(transaction)
+
+The `/deposit_indicative_quote` endpoint allows a wallet to get deposit information from an anchor when the user intends to make a conversion between non-equivalent tokens using an [indicative quote](https://www.investopedia.com/terms/i/indicativequote.asp). With this endpoint, a user has all the information needed to initiate a deposit and it also lets the anchor specify additional information (if desired) that the user must submit via [SEP-12](sep-0012.md).
+
+### Request
+
+```curl
+GET TRANSFER_SERVER/deposit_indicative_quote
+```
+
+Request Parameters:
+
+Name | Type | Description
+-----|------|------------
+`asset_code` | string | The code of the **on-chain** asset the user wants to get from the Anchor after doing an off-chain deposit. The value passed must match one of the codes listed in the [/info](#info) response's `deposit_indicative_quote` object.
+`source_asset` | string | The **off-chain** asset the Anchor will receive from the user. The value must match one of the `asset` values included in a [`SEP-38 GET /prices?buy_asset=stellar:<asset_code>:<asset_issuer>`](sep-0038.md#get-prices) response using [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
+`account` | `G...` string | The stellar account ID of the user that wants to deposit. This is where the asset token will be sent.
+`memo_type` | string | (optional) Type of memo that anchor should attach to the Stellar payment transaction, one of `text`, `id` or `hash`.
+`memo` | string | (optional) Value of memo to attach to transaction, for `hash` this should be base64-encoded.
+`email_address` | string | (optional) Email address of depositor. If desired, an anchor can use this to send email updates to the user about the deposit.
+`type` | string | (optional) Type of deposit. If the anchor supports multiple deposit methods (e.g. `SEPA` or `SWIFT`), the wallet should specify `type`. This field may be necessary for the anchor to determine which KYC fields to collect.
+`wallet_name` | string | (optional) In communications / pages about the deposit, anchor should display the wallet name to the user to explain where funds are going.
+`wallet_url` | string | (optional) Anchor should link to this when notifying the user that the transaction has completed.
+`lang` | string | (optional) Defaults to `en`. Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1). `error` fields in the response should be in this language.
+`on_change_callback` | string | (optional) A URL that the anchor should `POST` a JSON message to when the `status` property of the transaction created as a result of this request changes. The JSON message should be identical to the response format for the [/transaction](#single-historical-transaction) endpoint.
+`amount` | string | (optional) The amount of the asset the user would like to deposit with the anchor. This field may be necessary for the anchor to determine what KYC information is necessary to collect.
```

Comment:
What units is `amount` in within the context of an indicative deposit? I think it needs to be in units of the `source_asset` since thats the asset held by the user, and also gives the anchor an opportunity to return 400 if the amount is too large or small.

This also has implications for the `max_amount` and `min_amount` attributes in the `/info` endpoint for the associated `deposit_indicative_quote` object.

Reply:
Actually, I think we should make `amount` required for the indicative endpoints. I think this is especially important for the deposit case because we don't have a great way of communicating source asset deposit limits, as mentioned in my comments in `/info`.